css/styles.css

```diff
@@ -0,0 +1,67 @@
+* {
```

Comment:
I would move these styles to the body tag, so that the nested tags will inherit from it.